signer/plugin_test.go

```diff
@@ -115,7 +115,7 @@ func (p *mockPlugin) GenerateSignature(ctx context.Context, req *proto.GenerateS
 		return &proto.GenerateSignatureResponse{
 			KeyID:            req.KeyID,
 			Signature:        invalidSignatureEnvelope,
-			SigningAlgorithm: string(sigAlg),
+			SigningAlgorithm: sigAlg,
```

Comment:
Is it a little **breaking change**? We may need to mention it in the release.

Reply:
Yes looks like it is a small breaking change.